README.md

```diff
@@ -58,15 +58,15 @@ git, I worked on github and gitlab. I tried using git guis but I prefer to work
 * Can you describe your workflow when you create a web page?
 * If you have 5 different stylesheets, how would you best integrate them into the site?
 * Can you describe the difference between progressive enhancement and graceful degradation? 
-
 * What is semantic html?  
 Semantic HTML is the use of HTML markup to reinforce the semantics, or meaning, of the information in webpages and web applications rather than merely to define its presentation or look. Semantic HTML is processed by traditional web browsers as well as by many other user agents. CSS is used to suggest its presentation to human users.  
 
 * How would you optimize a website's assets/resources?  
 Optimalize images, bundle files, use uglifyjs and minification, use svg, css sprites, serve cdn resources depending on distance and response time from user, cache some files, dont load whole huge libraries if you only use one function from them  
 
-* How many resources will a browser download from a given domain at a time?
-  * What are the exceptions?
+* How many resources will a browser download from a given domain at a time? 
+It depends on a browser, older ones like IE6 - 2, newer ones 6-8.  
```

Comment:
Nie różni się, tylko polska jest bardziej rozwinięta. Bo to było dobrze zrobione w odpowiedziach od Ciebie a nie chciało mi się tłumaczyć z pl na angielski rozszerzenia odpowiedzi które uznałem za zbędne